sparse/coo/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ._core import COO
+from ._elemwise import elemwise
+from ._common import tensordot, dot, concatenate, stack, triu, tril, where, \
+    nansum, nanprod, nanmin, nanmax, nanreduce
```

Comment:
I recommend keeping these module names public.  I believe that private modules are somewhat rare.